pkg/distributor/distributor_retry_test.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package distributor
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRetryConfig_Validate(t *testing.T) {
+	t.Parallel()
+	tests := map[string]struct {
+		cfg         RetryConfig
+		expectedErr error
+	}{
+		"should pass with default config": {
```

Comment:
i'm not sure what other tests do, but to be sure the default passes you can add another test which uses `flagext.DefaultValues()` in case the defaults change. Not an important change though